src/components/NavBar/NavigationBar.tsx

```diff
@@ -35,24 +46,44 @@ function NavigationBar(): JSX.Element {
     setIsNavVisible(!isNavVisible);
   }
 
-  const {t} = useTranslation('menu');
+  /** Toggle the language of the website between English and Spanish. Close menu bar after toggling if on mobile. */
+  function toggleLanguage(): void {
+    i18n.changeLanguage(i18n.language === 'en' ? 'es' : 'en');
+    if (isMobile) {
+      toggleNav();
+    }
+  }
+
+  const {t, i18n} = useTranslation('menu');
   return (
     <div>
       <header className="Navigation_Header">
-        {}
         {/* Logo in Spanish and English are the same */}
         <img src={cehatLogo} className="Logo" alt="Logo" />
-        {/* CEHAT logo */}
         {isNavVisible && (
-          <nav className="Nav">
+          <nav>
             <a href="/">{t('home')}</a>
             <a href="/health">{t('healthInfo')}</a>
             <a href="/about">{t('about')}</a>
             <a href="/admin">{t('admin')}</a>
+            <button id="changeLanguage" onClick={toggleLanguage}>
+              {/* Display globe on desktop, but text in menu bar on mobile */}
+              {isMobile ? (
+                <span>
+                  <FaGlobeAmericas id="mobileGlobe" title={t('globeIcon')} />{' '}
```

Comment:
If possible, add some padding or something such that the text and icon have the same horizontal axis